src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -51,24 +50,24 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) =>
   })
   const [evaluatedParameters, setEvaluatedParameters] = useState({})
 
-  // This value prevents the plugin component from rendering until parameters have been evaluated, otherwise React throws an error when trying to pass an Object in as a prop value
-  const [parametersReady, setParametersReady] = useState(false)
-
   const { ApplicationView, config }: PluginComponents = pluginProvider.getPluginElement(pluginCode)
 
-  const dynamicParameters = config?.dynamicParameters
-  const dynamicExpressions =
-    dynamicParameters && extractDynamicExpressions(dynamicParameters, parameters)
+  const parameterLoadingValues = config?.parameterLoadingValues
+  const [simpleParameters, parameterExpressions] = buildParameters(
```

Comment:
any reason for using array vs object ?

Reply:
No particular reason, I just like this style for functions that return multiple values. I think it's more elegant for simple variables, but that's just personal taste.

(Although it would seem React has similar thoughts: `[ value, setValue ] = useState()` ;) )

Reply:
sure, all good with personal prefs.

react uses it to allow you to name it to whatever you want with minimum syntax